### PR TITLE
Added code of conduct and contributing btns. 

### LINKED ISSUE #54: Add Instruction to take a look at Code of Conduct and Contribution Guidelines
Add Instruction to take a look at Code of Conduct and Contribution Guidelines on getmein web page, email sent to new member and README too.

### PR DESCRIPTION
**Pull request by nlok5923**
Fixes #54

### Description

Added button to take a look at contributing guidelines and code of conduct.

###  Screenshot Before Change:
Screenshot (1344)

### Screenshot After Change:
Screenshot (1343)

### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Non Functional Requirement
- [ ] Follows the code style of this project.
- [ ] Tests Cover Changes
- [x] I have performed a self-review of my own code
- [ ] All new and existing tests passed.
- [ ] Documentation

### CODE REVIEW

**Review by aashutoshrathi — APPROVED** [review]
Code Looks Good, can you give a static deployment for us to quick check things?

**Comment by nlok5923** [comment]
Sir please have a look to the site : https://nlok5923-getmein-web-1.glitch.me/

**Review by aashutoshrathi — CHANGES_REQUESTED** [review]
Can we fit both buttons in the same row, also keep in mind to use 100vh and there should be no scroll bar on this app

**Comment by nlok5923** [comment]
okay sir working on it.

File: views/index.html [hunk 1]
```diff
@@ -97,13 +97,24 @@
           <button class="uk-button uk-margin-top uk-border-pill
             uk-button-default"
             type="submit" onclick="showToast()">Submit</button>
+           <br><br>
```
Comment (aashutoshrathi) [inline views/index.html]: Use `uk-margin-top` in element below it instead `<br/>`

File: views/index.html [hunk 2]
```diff
@@ -97,13 +97,24 @@
...
+            <span class="uk-text-success uk-text-meta uk-text-center uk-margin">Also take a look at </span>
+            <br>
+            <a href="https://github.com/iiitv/getmein-web/blob/master/CODE_OF_CONDUCT.md">
+            <button class="uk-button uk-margin-top uk-border-pill
```
Comment (aashutoshrathi) [inline views/index.html:104]: You are using `uk-button` the wrong way here.
When we have links, we should approach it like this.
```html
<a class="uk-button uk-margin-top uk-border-pill uk-button-default" 
    href="https://github.com/iiitv/getmein-web/blob/master/CODE_OF_CONDUCT.md"
    Code of Conduct
</a>
```

File: views/index.html [hunk 3]
```diff
@@ -97,13 +97,24 @@
...
+            uk-button-default">CODE OF CONDUCT</button></a>
+            <br>
+            </a>
+            <a href="https://github.com/iiitv/getmein-web/blob/master/CONTRIBUTING.md">
+            <button class="uk-button uk-margin-top uk-border-pill
```
Comment (aashutoshrathi) [inline views/index.html:109]: Same here, don't use the `button` element directly.

**Review by aashutoshrathi — CHANGES_REQUESTED** [review]

File: public/style.css [hunk 4]
```diff
@@ -12,7 +12,7 @@
 body {
   margin: 25px;
   overflow-y: scroll;
-  position: fixed;
+  position:fixed;
```
Comment (aashutoshrathi) [inline public/style.css:15]: Why is space here removed?

**Comment by aashutoshrathi** [comment]
Also, @nlok5923 Your commit messages indicate you are using GitHub Web GUI to edit files.
Please understand that we highly discourage this as it skips many layers of checks, please fork, clone and change files in editor locally, so that you automatically get styling fixes (at least).
cc: @iiitv/project-maintainers

**Comment by anshumanv** [comment]
Yep, please write meaningful commit messages/descriptions to indicate what the change does. GUI 👎

**Comment by nlok5923** [comment]
okay sir sorry for using Web ui i will create a new pull request.

**Pull request closed without merging**